OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -1670,3 +1677,20 @@ void testExternalForce()
             ASSERT_EQUAL(def, val, 10*accuracy);
     }
 }
+
+void testSerializeDeserialize() {
```

Comment:
Yes. So changing the model file will not make the test better. And I don't think we can load the file in the *old* style (as if version number on the file is the latest version) preserving *isDisabled* so it can be checked against *appliesForce*.